### PR TITLE
[ACR-1623] Add the HEIF description

### PR DESCRIPTION
**Pull request by jiyong2**
- ACR: ACR-1623

### CODE REVIEW

File: docs/application/native/guides/multimedia/image-edit.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The main features of the Image Util API include:
   You can [decode images](#decode), [encode them](#encode), and [encode animation](#animation) with the following formats:
 
   - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
-  - Input image formats for decoding: JPEG, PNG, GIF, BMP, and WEBP
+  - Input image formats for decoding: JPEG, PNG, GIF, BMP, WEBP and HEIF
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-edit.md:29]:
```suggestion
  - Input image formats for decoding: JPEG, PNG, GIF, BMP, WEBP, and HEIF
```

File: docs/application/native/guides/multimedia/image-edit.md [hunk 2]
```diff
@@ -386,7 +386,7 @@ To crop a media packet:
 <a name="decode"></a>
 ## Decoding from a File or Memory
 
-To decode a JPEG, PNG, GIF, or BMP image:
+To decode a JPEG, PNG, GIF, BMP, WEBP or HEIF image:
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-edit.md:389]:
```suggestion
To decode a JPEG, PNG, GIF, BMP, WEBP, or HEIF image:
```

File: docs/application/native/guides/multimedia/image-edit.md [hunk 3]
```diff
@@ -410,7 +410,7 @@ To decode a JPEG, PNG, GIF, or BMP image:
    ```
 
    > [!NOTE]
-   > Due to the decoder limitations, the color space setting is only supported for decoding the JPEG and the WEBP images.
+   > Due to the decoder limitations, the color space setting is only supported for decoding the JPEG, the WEBP and the HEIF images.
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-edit.md:413]:
```suggestion
   > Due to the decoder limitations, the color space setting is only supported for decoding the JPEG, the WEBP, and the HEIF images.
```

**Review by omar-srbd — COMMENTED** [review]
@jiyong2, please incorporate the changes as per suggestion.
Thank you.

**Comment by omar-srbd** [comment]
@mijin85cho, Hello.
During the check of the file (docs/application/native/guides/multimedia/image-edit.md) on this PR we found this link that leads to a different site than the one it was intended to. I think this might have happened due to the expiration of the domain.
For your reference I am adding the links below:
Current link: http://www.fourcc.org/yuv.php
image

Where it should lead to: https://web.archive.org/web/20220830193330/https://fourcc.org/

**Comment by mijin85cho** [comment]
> @mijin85cho, Hello. During the check of the file (docs/application/native/guides/multimedia/image-edit.md) on this PR we found this link that leads to a different site than the one it was intended to. I think this might have happened due to the expiration of the domain. For your reference I am adding the links below: Current link: http://www.fourcc.org/yuv.php image
> 
> Where it should lead to: https://web.archive.org/web/20220830193330/https://fourcc.org/

Hello, @jiyong2

it seem a reference link in this guide is outdated. 
Would you please suggest another link instead if possible?
Or, how about removing the line 69?

File: docs/application/native/guides/multimedia/image-edit.md (hunk 1 above)
Comment (jiyong2) [reply docs/application/native/guides/multimedia/image-edit.md]: Thank you for your kind comment.
I added missing comma.

File: docs/application/native/guides/multimedia/image-edit.md (hunk 2 above)
Comment (jiyong2) [reply docs/application/native/guides/multimedia/image-edit.md]: Thank you. I added missing comma.

File: docs/application/native/guides/multimedia/image-edit.md (hunk 3 above)
Comment (jiyong2) [reply docs/application/native/guides/multimedia/image-edit.md]: Thank you. I added it.

**Comment by jiyong2** [comment]
Thank you for the comment of the outdated link.
Developers can easily get that information from various blogs and sites. Also, even if you do not know the information on the link, there is no problem with using API. Because it was information for difficult use, I thought it was unnecessary information in the guide and removed the link.

**Review by omar-srbd — APPROVED** [review]
LGTM.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Pull request merged**